### DISCUSSION TITLE
convert it to a regular latitude-longitude grid

### DISCUSSION THREAD

**Discussion by liujieljlj** [blaylockbk/goes2go#76] [Q&A]
I want to use ABI-L1b data, but this data is in a geostationary projection. When I use it, I need to convert it to a regular latitude-longitude grid. How should I handle this?

**Comment by blaylockbk** [comment]
This is an old notebook that might be outdated, but it might point you in the right direction. 

https://github.com/blaylockbk/goes2go/blob/main/latlon_grids/create_standard_GOES_latlon_grids.ipynb

**Comment by liujieljlj** [comment]
收到

**Comment by blaylockbk** [comment]
Found this
https://github.com/uba/goes-latlon

**Comment by liujieljlj** [comment]
收到

**Comment by blaylockbk** [comment]
Duplicate #30
